examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/WildcardFragment.kt

```diff
@@ -182,6 +208,35 @@ class WildcardFragment : Fragment() {
     }
   }
 
+  private suspend fun write(writeValueType: String, writeValue: String, dataVersion: Int?, timedRequestTimeoutMs: Int, imTimeoutMs: Int) {
+    val endpointId = getChipPathIdForText(binding.endpointIdEd.text.toString())
+    val clusterId = getChipPathIdForText(binding.clusterIdEd.text.toString())
+    val attributeId = getChipPathIdForText(binding.attributeIdEd.text.toString())
+    val tlvWriter = TlvWriter()
+    val values = writeValue.split(",")
```

Comment:
is a `.trim()` useful here as well in case it is separated by both a space as well as a `','`?

i.e. "foo, bar , baz"

Reply:
Yes.